lib/internal/streams/destroy.js

```diff
@@ -103,8 +103,12 @@ function errorOrDestroy(stream, err) {
 
   if ((rState && rState.autoDestroy) || (wState && wState.autoDestroy))
     stream.destroy(err);
-  else
+  else {
```

Comment:
Since we're switching to multi-line `else` here, can you also add braces for the `if` statement above as well?